internal/k8s/deployments.go

```diff
@@ -159,9 +160,9 @@ func TruncateString(s string, n int) string {
 }
 
 func CleanStringForDNS(s string) string {
-	// Keep only letters, numbers, and dashes.
-	re := regexp.MustCompile(`[^a-zA-Z0-9-]+`)
-	return re.ReplaceAllString(s, K8sResourceNameSeparator)
+	// Keep only letters, numbers, dashes, and dots
+	re := regexp.MustCompile(`[^a-zA-Z0-9-.]+`)
```

Comment:
I specifically stopped allowing dots in the pod names because it caused there to be warnings on every reconcile loop. See https://github.com/temporalio/temporal-worker-controller/pull/135

I think we should have no dots in pod names, but allow all characters in the build ids stored in temporal

Reply:
Thanks, I messed this up when merging conflicts.